assets/src/karma-fixture/init.js

```diff
@@ -369,6 +369,11 @@ beforeEach(async () => {
     margin: 0;
   `;
   document.body.appendChild(rootEl);
+  document.body.cssText = `
+    height: 100vh;
+    width: 100vw;
+    margin: 0;
+  `;
```

Comment:
As can be seen a few lines above, the actual editor app is embedded in a fixed-position div, so the body height is never updated to incorporate the editor. This updates the document height to just be the entire window regardless of the actual editor app size. It's not perfect, but it'll make the modal calculations work correctly at least.

Reply:
Oh, good idea! Will add that as a comment as well!